release/release.go

```diff
@@ -15,6 +15,28 @@ import (
 	"github.com/mongodb/mongo-tools/release/platform"
 )
 
+// The msi msiUpgradeCode must be updated when the minor version changes.
+var msiUpgradeCode = "56c0fda6-289a-4fd0-a539-6711864146ba"
```

Comment:
Same as you did for the filepaths, can you move this into `buildMSI()` since it's msi-specific?